e2e/Makefile

```diff
@@ -22,8 +20,6 @@ GINKGO := $(BINDIR)/ginkgo-$(GINKGO_VERSION)
 MINIKUBE_HOME = $(BINDIR)
 export MINIKUBE_HOME
 
-export TEST_KUBERNETES_VERSION=$(shell echo $(KUBERNETES_VERSION) | grep -o '^[0-9]*\.[0-9]*')
```

Comment:
This environment variable is not used as of b6c1cea5821504d776489aa5038afa16cd862c6e